nodeJS/apis/api_security.md

```diff
@@ -22,14 +22,16 @@ This strategy, while particularly useful with APIs can be used with a traditiona
 <div class="lesson-content__panel" markdown="1">
 
 1. [This video](https://www.youtube.com/watch?v=7nafaH9SddU) is a great resource that explains everything you need to know about creating and verifying JSON Web Tokens.
-2. [This article](https://dev.to/_arpy/learn-using-jwt-with-passport-authentication-22n8) covers setting up JSON Web Tokens within the PassportJS system that you should already be familiar with. In this case you don't _really_ need Passport, but it does handle quite a bit of stuff that the guy in the video set up manually.
+2. [This article](https://dev.to/_arpy/learn-using-jwt-with-passport-authentication-22n8) covers setting up JSON Web Tokens within the PassportJS system that you should already be familiar with. In this case you don't _really_ need Passport, but it does handle quite a bit of stuff that the guy in the video set up manually. NOTE: There is some typos in the code blocks provided in the article, its good exercise to code along and spot them ;). Aswell, some function are now deprecated and this needs some extra reading of documentation in order to make it work.
+3. [This video](https://www.youtube.com/watch?v=JdGOb7AxUo0) is a good counterbalance in the use of JWT versus use of Session Cookie. It states some facts and is a good way to comprehend a larger scope of the use of JWT for authentication and some of the danger coming along.
+
 </div>
 
 
 ### Additional Resources
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
-- [This article](https://laptrinhx.com/a-practical-guide-for-jwt-authentication-using-node-js-and-express-917791379/) goes even further in depth than the one that we posted above. It provides a little greater understanding but is harder to follow.
+- [This article](https://laptrinhx.com/a-practical-guide-for-jwt-authentication-using-node-js-and-express-917791379/) goes even further in depth than the one that we posted above. It provides a little greater understanding but is harder to follow. 
```

Comment:
try to avoid making these kinds of changes to the content.  Remove this space, please.

Reply:
Sorry about that..